airbyte-server/src/test/java/io/airbyte/server/handlers/ArchiveHandlerTest.java

```diff
@@ -295,17 +286,17 @@ private void setupTestData(final UUID workspaceId) throws JsonValidationExceptio
     configPersistence.writeConfig(ConfigSchema.SOURCE_CONNECTION, sourceid.toString(), new SourceConnection()
         .withSourceId(sourceid)
         .withWorkspaceId(workspaceId)
-        .withSourceDefinitionId(configRepository.listStandardSourceDefinitions().get(0).getSourceDefinitionId())
+        .withSourceDefinitionId(UUID.fromString("ef69ef6e-aa7f-4af1-a01d-ef775033524e")) // GitHub source definition
         .withName("test-source")
-        .withConfiguration(Jsons.emptyObject())
+        .withConfiguration(Jsons.jsonNode(ImmutableMap.of("start_date", "2021-03-01T00:00:00Z", "repository", "airbytehq/airbyte")))
```

Comment:
Made this change because passing in an empty Configuration no longer works, since the test uses specs in the seed yaml files to validate the Configuration json. 

I explored trying to make the YamlSeedConfigPersistence instance in the test set the specs to empty so that this test would continue to pass as-is, but that ended up being much more complicated than just creating a valid configuration like I did here. The downside of this approach is that if the required fields of the GitHub or BigQuery specs change, then the test will fail, but that seems like an okay trade-off to a more complex change to these tests to make specs of the seed connector definitions empty.